### PR TITLE
Fix ci jobs when unify version

### PR DESCRIPTION
**Pull request by Fuerback**
This PR fixes the `Chaincode Template Lint` job when `unifyVersions.js` is executed.

It also updates the `unifyVersions.js` to update a few files (e.g. licenses, lock, oclif manifest) after update the `package.json` files.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 name: CI
 on:
   push:
-    branches: ['**']
```
Comment (dzikowski) [inline .github/workflows/ci.yml:4]: is it safe to remove it?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -122,65 +121,63 @@ jobs:
...
+        run: |
+          $package = Get-Content -Path test-project/package.json -Raw
+          if ($package -match '@gala-chain/test-project') {
+            Write-Host "package.json contains '@gala-chain/test-project'"
+          } else {
+            Write-Host "package.json does not contain '@gala-chain/test-project'"
+            exit 1
+          }
 
-  # chaincode-template-lint:
```
Comment (dzikowski) [inline .github/workflows/ci.yml:156]: comment to be removed

**Review by dzikowski — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Fuerback) [reply .github/workflows/ci.yml:4]: It's duplicating the job executions after we added the `pull_request`.
My understanding is that we should choose which one we want to keep, for every branch (and it includes pull requests) or only when a pull request is created.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (Fuerback) [reply .github/workflows/ci.yml:156]: There is no more comment line in ci.yml.

**Review by herr-bubu — APPROVED** [review]

**Pull request merged**